src/Radicle/Internal/PrimFns.hs

```diff
@@ -709,8 +709,10 @@ purePrimFns = fromList $ allDocs $
           let allMod = fromEnv e
           toImport <- case v_ of
                 Just vs -> do
-                  is <- Protolude.toList <$> traverse isAtom vs ?? toLangError (OtherError "import: must be given a vector of symbols to import")
-                  pure $ Map.restrictKeys allMod (Set.fromList is)
+                  is <- Set.fromList <$> Protolude.toList <$> traverse isAtom vs ?? toLangError (OtherError "import: must be given a vector of symbols to import")
```

Comment:
First converting to a list and then to a set is awkward, but I couldn't find any builtin to convert Seq to Set directly.

Reply:
I think that's fine